src/runner/configuration.rs

```diff
@@ -0,0 +1,16 @@
+
+pub struct Configuration {
+    pub parallel: bool,
+}
+
+impl Configuration {
+    pub fn parallel(self, parallel: bool) -> Self {
+        Configuration { parallel: parallel }
```

Comment:
I'm using the `derive_builder` for Configuration now to keep things clean and future-proof.

After all this method was little more than an attempt at a poor man's builder pattern.